#!/usr/bin/env python
PACKAGE = "transparent_object_reconstruction"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sample_dist", double_t, 0, "Point sample distance in occlusion cone creation", .01, 0, 1)
gen.add("random_noise", bool_t, 0, "Add uniformly distributed random noise to sampling dist for each ray", False)
gen.add("std_dev_factor", double_t, 0, "Factor to influence standard deviance for norm distribution (std_dev = sample_dist * std_dev_factor)", .1, 0, 1)

exit(gen.generate(PACKAGE, "transparent_object_reconstruction", "CreateRays"))
